components/vue/madata-vue.js

```diff
@@ -142,6 +142,9 @@ const MaData = {
 		async login (o) {
 			this.inProgress = "Logging in...";
 			await this.backend.login(o);
+			if (!this.dataLoaded) {
+				await this.load();
```

Comment:
This would call `this.load()` when `login()` is called, even if nothing has called `load()` before. Though maybe that's ok, since the plugin calls `load()` immediately?

Reply:
This was precisely my thought: every time `login()` might be invoked, it's invoked after `load()`.